tests/unit_tests/layout/test_layout_array.py

```diff
@@ -232,3 +229,59 @@ def test_altitude_from_corsika_z(layout_center_data_dict, corsika_telescope_data
     )
     with pytest.raises(TypeError):
         layout._altitude_from_corsika_z(5.0, None, "LST-01")
+
+
+def test_include_radius_into_telescope_table(layout_array_instance, telescope_test_file):
```

Comment:
Since we decided to keep this function to maintain the plot function independent of the metadata and DB, I am closing here, but feel free to reopen it if you want to further discuss it

Reply:
OK in that case allow me to review it then. 

I think you can simply use a list below, no need for a numpy array
```
values_from_file = np.array([20.190000534057617, -352.4599914550781, 62.29999923706055, 9.6])
```

I think it would be nicer to run on the actual values of the two lists with zip rather than with enumerate in the line below
```
for key_step, _ in enumerate(keys):
```

Why do you need the `[0]` in
```
telescope_table_with_radius[mst_10_index][keys[key_step]].value[0]
```

Lastly, you shouldn't assert floats with `==` but use `pytest.approx`